packages/@lwc/integration-karma/test-hydration/attributes/falsy-mismatch/index.spec.js

```diff
@@ -31,8 +31,8 @@ export default {
         TestUtils.expectConsoleCallsDev(consoleCalls, {
             error: [],
             warn: [
-                'Mismatch hydrating element <div>: attribute "data-foo" has different values, expected "undefined" but found null',
-                'Mismatch hydrating element <div>: attribute "data-foo" has different values, expected "null" but found null',
+                'Hydration attribute mismatch on: DIV - rendered on server: data-foo=null - expected on client: data-foo="undefined"',
+                'Hydration attribute mismatch on: DIV - rendered on server: data-foo=null - expected on client: data-foo="null"',
```

Comment:
This is much more legible than before, but again I think `<div>` is better than `DIV` here.